core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2178,22 +2179,24 @@ class WorkspaceService(protected val userInfo: UserInfo,
     )
   }
 
-  private def syncPolicies(workspaceId: String, policyEmailsByName: Map[SamResourcePolicyName, WorkbenchEmail], workspaceRequest: WorkspaceRequest, parentSpan: Span) = {
+  private def syncPolicies(workspaceId: String, policyEmailsByName: Map[SamResourcePolicyName, WorkbenchEmail], workspaceRequest: WorkspaceRequest, parentSpan: Span): Future[List[Any]] = {
     traceWithParent("traversePolicies", parentSpan) (s1 =>
-      Future.traverse(policyEmailsByName.keys) { policyName =>
+      // This used to be Future.traverse but that runs all the futures eagerly and concurrently. This led to some perf
+      // issues, hitting the akka subscription timeout. We switched to use cats traverse to make these calls serially.
+      policyEmailsByName.keys.toList.traverse { policyName =>
         if (policyName == SamWorkspacePolicyNames.projectOwner && workspaceRequest.authorizationDomain.getOrElse(Set.empty).isEmpty) {
           // when there isn't an auth domain, we will use the billing project admin policy email directly on workspace
           // resources instead of synching an extra group. This helps to keep the number of google groups a user is in below
           // the limit of 2000
-          Future.successful(())
+          IO.fromFuture(IO(Future.successful(())))
         } else if (WorkspaceAccessLevels.withPolicyName(policyName.value).isDefined || policyName == SamWorkspacePolicyNames.canCompute) {
           // only sync policies that have corresponding WorkspaceAccessLevels to google because only those are
           // granted bucket access (and thus need a google group)
-          traceWithParent(s"syncPolicy-${policyName}", s1)(_ => samDAO.syncPolicyToGoogle(SamResourceTypeNames.workspace, workspaceId, policyName))
+          IO.fromFuture(IO(traceWithParent(s"syncPolicy-${policyName}", s1)(_ => samDAO.syncPolicyToGoogle(SamResourceTypeNames.workspace, workspaceId, policyName))))
         } else {
-          Future.successful(())
+          IO.fromFuture(IO(Future.successful(())))
         }
-      }
+      }.unsafeToFuture()
```

Comment:
I think all this can be simplified into one `Applicative[Future].whenA` call (or whatever the logic is, it's too confusing)
```scala
// why do we not have these predicates???
def isProjectOwner(policy: SamResourcePolicyName): Boolean = 
  policy == SamWorkspacePolicyNames.projectOwner

def isCanCompute(policy: SamResourcePolicyName): Boolean = 
  policy == SamWorkspacePolicyNames.canCompute

def hasAuthorizationDomain(request: WorkspaceRequest): Boolean =
  request.authorizationDomain match {
    case Some(xs) => !xs.isEmpty
    case None => False
  }

...

Applicative[Future].whenA(
  !isProjectOwner(policyName) && 
  hasAuthorizationDomain(workspaceRequest) &&
  (WorkspaceAccessLevels.withPolicyName(policyName.value).isDefined ||  isCanCompute(policyName)),
  traceWithParent(s"syncPolicy-${policyName}", s1)(_ => 
    samDAO.syncPolicyToGoogle(SamResourceTypeNames.workspace, workspaceId, policyName)
  )
)
```